modules/viz/src/vtk/vtkCocoaInteractorFix.mm

```diff
@@ -160,7 +160,7 @@ - (void)windowWillClose:(NSNotification*)aNotification
 
 //----------------------------------------------------------------------------
 
-#if VTK_MAJOR_VERSION >= 6 && VTK_MINOR_VERSION  >=2
+#if VTK_MAJOR_VERSION >= 6 && VTK_MINOR_VERSION >=2 || VTK_MAJOR_VERSION >=7
```

Comment:
Well, I am not sure if I agree on "more readable", but you decide : )

Do you want me to fix it?